Sources/Extensions/DateFormatter.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 extension DateFormatter {
-    static let rfc3339: DateFormatter = {
+    public static let rfc3339: DateFormatter = {
```

Comment:
ah, I did that to decode an experience for testing in ViewController.swift.  Couldn't set up a JSONDecoder properly otherwise.

Naturally can revert this bit after the poll project, too.